spec/requests/dashboard_spec.rb

```diff
@@ -268,6 +268,14 @@
         get "/dashboard/analytics"
         expect(response.body).to include("Analytics")
       end
+
+      it "page always contain back to dashboard button" do
+        sign_in user
+        get "/dashboard/analytics"
```

Comment:
Tried to use visit here instead of get but got undefined method.

Reply:
`visit` is for system tests, this is a request spec.